gnd/src/main/java/com/google/android/gnd/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -141,7 +143,8 @@ private Completable insertOrUpdateField(String formId, Element.Type elementType,
             Observable.just(field)
                 .filter(__ -> field.getMultipleChoice() != null)
                 .flatMapCompletable(
-                    __ -> insertOrUpdateMultipleChoice(field.getId(), field.getMultipleChoice())))
+                    __ -> insertOrUpdateMultipleChoice(field.getId(),
+                        Preconditions.checkNotNull(field.getMultipleChoice()))))
```

Comment:
Can we static import `Preconditions` throughout to make these more inline friendly and reduce line-wrapping?

Reply:
Ah right, sorry I get what you mean now. I'll update throughout.